src/main/java/org/group23/QuestionRepository.java

```diff
@@ -0,0 +1,8 @@
+package org.group23;
+
+import org.springframework.data.repository.CrudRepository;
+
+public interface QuestionRepository extends CrudRepository<Question, Long> {
+
+    Question findByid(Long id);
```

Comment:
Is this the correct case? as opposed to findById

Reply:
Replied in other comment.